### PR TITLE
Adapt Sentence Transformer model to BertForSeqClassification

### PR DESCRIPTION
**Pull request by caronzh03**
- Change BertPooler to use mean pooling
- Retrofit a ST bert model to HF's BertForSeqClassification
- Print logits during inference

### CODE REVIEW

File: triton_backend/all_models/llmapi/tensorrt_llm/1/model.yaml
Comment (nvrohanv) [inline triton_backend/all_models/llmapi/tensorrt_llm/1/model.yaml:1]: same as quickstart.py and model.py is this a file you want to change?

File: examples/pytorch/sbert.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    num_labels=192,  # Set number of classes
+    hidden_dropout_prob=bert_model.config.hidden_dropout_prob,
+    attention_probs_dropout_prob=bert_model.config.attention_probs_dropout_prob
+)
+
+# 4. Create new classification model
+classification_model = BertForSequenceClassification(config)
+
+# 5. Copy weights
+classification_model.bert.load_state_dict(bert_model.state_dict(), strict=False)
```
Comment (nvrohanv) [inline examples/pytorch/sbert.py:42]: This might be a stupid question but I'm a bit confused why we have to copy the weights when in line 31 we created a config that had the model path and then in line 39 created classification_model from that config.

**Review by nvrohanv — COMMENTED** [review]
The approach is good, but new files need to be created for the examples instead of permanently changing things like quickstart.py.
